nugridpy/data_plot.py

```diff
@@ -3862,8 +3862,8 @@ def elemental_abund(self,cycle,zrange=[1,15],ylim=[0,0],title_items=None,
                 'Z': charge number
                 '[X/Fe]': metallicity
         zchi2 : list, optional
-            A 1x2 array containing the lower and upper atomic number
-            limit for chi2 test when pin_filename != None
+            A 1x2 array containing atomic numbers of the elements
+            for which chi2 test is done when plotType == 'PPN' and pin_filename != None
```

Comment:
Should this be when `ref_filename != None` instead? That's when `chi2` is defined... or should `chi2` be defined one if statement deeper than it currently is?

Reply:
For the chi2 method to work we need observed abundances to compare predicted abundances with, and the former are in pin_filename. I think...

Reply:
Okay, that makes sense. But then why do we define and return `chi2 = 0` outside of the `if pin_filename != None` statement? Shouldn't we do that within the if statement? Or do we want to return `chi2 = 0` in every case when `pin_filename == None` ?

Reply:
This is sort of nitpicky though, it doesn't actually make any difference, it's just one extra unnecessary float being returned if the `if` statement doesn't trigger, and that's negligible. Otherwise I have run notebooks that use these functions and things seem to run fine...